### DISCUSSION TITLE
How to resolve Okta certificate pinning issues?

### DISCUSSION THREAD

**Discussion by changsin** [mitmproxy/mitmproxy#6553] [General]
We were using mitmproxy to build an app to monitor HTTP/S traffic, but one of our users encountered not being able to use okta because of its certificate pinning. We tried to do ignore_hosts of okta domains but it still did not work. Has anyone experience similar issues and is there any guideline for certificate pinning issues in general? Thanks,

**Comment by rosydawn6** [comment]
I have had good success using ignore_hosts for similar scenarios.  You need to make sure to find & ignore all of them as some may have unexpected domain names.

**Reply by changsin** [reply]
Thanks. That's what we ended up doing as well by adding *.okta.com in the ignore_hosts list.
